ts/theme/variables.ts

```diff
@@ -58,7 +58,7 @@ const customVariables = Object.assign(materialVariables, {
     return this.fontSizeBase * 1.125;
   },
   get fontSize5(): number {
-    return this.fontSizeBase * 1.5;
+    return this.fontSizeBase * 10.5;
```

Comment:
multiplicator should be something in between `fontSize4` and `fontSize6`

Reply:
whoops, I was messing around to test things and forgot to change this back!